### ISSUE TITLE
Sharing edge between multiple node pairs?

### ISSUE THREAD

**Issue by SiftingSands** [arangodb/arangodb#16206]
## My Environment

* __ArangoDB Version__:        3.9.1
* __Deployment Mode__:      Single Server
* __Deployment Strategy__:  ArangoDB Starter in Docker
* __Configuration__:              Using the default rocksDB storage engine
* __Infrastructure__:               own
* __Operating System__:        RHEL 7
* __Total RAM in your machine__:        1TB
* __Disks in use__:        SSD and network storage devices
* __Used Package__:              Docker/Singularity

## Question

I've got 10,000s of nodes that are connected by edges with the same "content", so I was wondering if arangoDB supported an edge containing multiple from/to pairs? Let me know if this is a moot point in terms of storage compression and/or query performance. It just seemed redundant at first glance, but maybe the "backend" does this optimization already. 

Thanks!

**Comment by dothebart** [comment]
the nature of JSON prohibits having the same key twice.
You can however have several edge documents with the same `_from/_to` combination - if that answers your question?

**Comment by SiftingSands** [comment]
In my current case, I wouldn't be referencing the same key more than once. I was thinking about having an edge that defined the connection between, let's say, nodes 1-2, 3-4, 5-6 and so on.

Giving this some more thought, I could create a separate collection of edges and reference those edges in a graph by their 'key' to reduce duplicate data being stored?

**Comment by dothebart** [comment]
so you want something spanning multiple edges without having a vertex?

**Comment by SiftingSands** [comment]
The edges would still connect between vertices with no dangling edge, but I'm not sure if that's what you meant.

Maybe this helps illustrate my intention :
Let's say I want to share edge content between multiple connected vertices.
Edge1 = {'date': '2022-05-02', 'high_five': True}
Edge2 = {'date': '2022-05-02', 'high_five': False}

Vertex 1 --Edge1--> Vertex 2
Vertex 3 --Edge1--> Vertex 4
Vertex 5 --Edge2-> Vertex 6
Vertex 7 --Edge2-> Vertex 8

**Comment by dothebart** [comment]
No, that's not possible like this. If you want to group edges in a way, you can do so by adding attributes qualifying that. 
so:
```
Edge 1A:
{
   _from: v1,
  _to:v2,
  group: "abc123"
}
Edge 1B:
{
  _from: v1,
  _to: v4,
  group: "abc123"
}
```
you would back that up with [vertex centric indices](https://www.arangodb.com/docs/stable/indexing-index-basics.html#vertex-centric-indexes)

**Comment by SiftingSands** [comment]
Sounds good. Thanks for confirming!
